### ISSUE TITLE
Improve UI/UX Accessibility for users

### ISSUE THREAD

**Issue by TundeAdekoya** [monum/wheel-of-kindness#2]
1) Provide alternative text for images: Alternative text (alt text) is a description of an image that is read by screen readers. It should be concise and accurately describe the content and function of the image.

2) Use semantic HTML elements to accurately describe the content they contain, main content of the document, sections content to separate documents, variations of the h tag to clarify the importance of the text.

3) Create Text hierarchy in the design.

4) Adequate margin and padding spaces to enhance both web and mobile experience  

5) Proper text line height for easy readability

**Comment by TundeAdekoya** [comment]
I'm interested in making these enhancements

**Comment by kartikaysaxena** [comment]
Can I be assigned this issue?

**Comment by aman225** [comment]
hey @TundeAdekoya can i you assign my this i would like to work on this.
